### DISCUSSION TITLE
History of watched anime

### DISCUSSION THREAD

**Discussion by an-dr32** [pystardust/ani-cli#915] [General] [answered]
Hey guys. I was wondering why the history of watched animes was removed. Or is it something that needs to be enabled? In previous iterations it was automatic but now I see no option anywhere. Anyone knows what happened there?

**Comment by Derisis13** [comment, marked as answer]
This happened only on major version releases, and it was recoverable manually as the history files are human-readable. The reason is that changing provider meant different history data we need to store.
